src/dstack/_internal/cli/services/configurators/run.py

```diff
@@ -30,11 +33,20 @@ def register(cls, parser: argparse.ArgumentParser):
         )
 
     @classmethod
-    def apply(cls, args: argparse.Namespace, conf: BaseConfiguration):
+    def apply(cls, args: argparse.Namespace, unknown: List[str], conf: BaseConfiguration):
         if args.envs:
             for k, v in args.envs:
                 conf.env[k] = v
 
+        cls.interpolate_list(conf.setup, unknown)
+
+    @classmethod
+    def interpolate_list(cls, value: List[str], unknown):
```

Comment:
This method interpolates only `run.args`, I believe it should be called `interpolate_args` or `interpolate_commands`

Reply:
It's both `setup` and `commands`